### DISCUSSION TITLE
Can't implement the Laravel Auth with this package

### DISCUSSION THREAD

**Discussion by itskawsar** [spatie/laravel-multitenancy#354] [Q&A] [answered]
The route is working fine, for example:
http://tenant1.dev/login
http://tenant2.dev/login
http://landlord.dev/login

But when I'm trying to login, I get the following message:
```
Illuminate\Database\QueryException
SQLSTATE[42S02]: Base table or view not found: 1146 Table 'dev_spatie_mt_landlord.users' doesn't exist
`select * from `users` where `email` = robel.america@example.net limit 1`
```

This means when I'm trying to login, the Auth trying to find user on landlord db instead of tenant db. I have checked the current tenant with the following code: `dd(Spatie\Multitenancy\Models\Tenant::current());` and it's showing the current tenant as expected.

I'm using PHP v8.0.11 and Laravel v9.7.0

I have installed the fresh Laravel and added `laravel/breeze` and `spatie/laravel-permission` packages. After that I have install the `spatie/laravel-multitenancy` package by following the official documentation.

Am I missed anything? Please help me.
Thanks in advance.

**Comment by masterix21** [comment, marked as answer]
Extend your user model and add to it the `UsesTenantConnection` trait.
